### DISCUSSION TITLE
Reduced rate of updates through NPM RSS feed

### DISCUSSION THREAD

**Discussion by ptelang** [community/community#135920] [npm] [answered]
Hello, We ingest NPM packages using the ossf/package-feeds project, which depends upon NPM RSS feed for newly published packages, or package updates. In last few days, we noticed a remarkable drop in the rate of new packages or package updates coming through the RSS feed. Has something changed on the NPM end that may cause this drop? Thanks.

**Comment by leobalter** [comment]
Thanks @ptelang! We identified that prevented updates in our RSS feed and deployed a fix this week. It should be resolved now.

**Reply by ptelang** [reply]
Hi @leobalter, We are still seeing very few (new and updated) packages in the RSS feed. So the issue does not appear to be fixed.

**Reply by ptelang** [reply]
Hi @leobalter, We started seeing a larger number of new and updated packages today (09/03). So the feed issue seems to be fixed. Can you please confirm if the issue is fully resolved from the NPM side? Really appreciate your help. Thanks.

**Reply by leobalter** [reply, marked as answer]
Thanks, @ptelang! I've been tracking this and it all seems stable now.

**Reply by ptelang** [reply]
Hi @leobalter, Thanks for the confirmation. In future, is there a way we can get notifications for such issues?
